examples/manageddisk_lustre/config.json

```diff
@@ -0,0 +1,257 @@
+    "headnode": {
+      "type": "vm",
+      "vm_type": "Standard_HB60rs",
+      "accelerated_networking": false,
+      "public_ip": true,
+      "image": "variables.hpcimage",
+      "subnet": "compute",
+      "tags": [
+        "disable-selinux",
+        "lfsclient",
+        "lfsazimport",
+        "localuser",
+        "pbsserver",
+        "loginnode",
+        "nfsserver"
+      ]
+    },
+    "compute": {
+      "type": "vmss",
+      "vm_type": "Standard_HB60rs",
+      "accelerated_networking": false,
+      "instances": 2,
+      "image": "variables.hpcimage",
+      "subnet": "compute",
+      "tags": [
+        "disable-selinux",
+        "lfsclient",
+        "localuser",
+        "pbsclient",
+        "nfsclient"
+      ]
+    },
+    "lfsmds": {
+      "type": "vm",
+      "vm_type": "Standard_D16s_v3",
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "storage_sku": "Premium_LRS",
+      "data_disks": [
+        4096
+      ],
+      "tags": [
+        "lfspkgs",
+        "lfsmaster",
+        "disable-selinux",
+        "lfsloganalytics"
+      ]
+    },
+    "lfsoss": {
+      "type": "vmss",
+      "vm_type": "Standard_D16s_v3",
+      "instances": 2,
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "storage_sku": "Premium_LRS",
+      "data_disks": [
+        1024,
+        1024,
+        1024,
+        1024,
+        1024,
+        1024
+      ],
+      "tags": [
+        "lfspkgs",
+        "lfsoss",
+        "disable-selinux",
+        "lfsloganalytics"
+      ]
+    },
+    "lfshsm": {
+      "type": "vmss",
+      "vm_type": "Standard_D16s_v3",
+      "instances": 1,
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "tags": [
+        "lfspkgs",
+        "lfshsm",
+        "disable-selinux"
+      ]
+    },
+    "lfssmb": {
+      "type": "vm",
+      "vm_type": "Standard_D16s_v3",
+      "instances": 1,
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "tags": [
+        "lfspkgs",
+        "lfsclient",
+        "lfssmb",
+        "disable-selinux",
+        "localuser"
+      ]
+    }
+  },
+  "install": [
+    {
+      "script": "disable-selinux.sh",
+      "tag": "disable-selinux",
+      "sudo": true
+    },
+    {
+      "script": "nfsserver.sh",
+      "tag": "nfsserver",
+      "sudo": true
+    },
+    {
+      "script": "nfsclient.sh",
+      "args": [
+        "$(<hostlists/tags/nfsserver)"
+      ],
+      "tag": "nfsclient",
+      "sudo": true
+    },
+    {
+      "script": "localuser.sh",
+      "args": [
+        "$(<hostlists/tags/nfsserver)"
+      ],
+      "tag": "localuser",
+      "sudo": true
+    },
+    {
+      "script": "create_raid0.sh",
+      "args": [
+        "/dev/md0",
+        "/dev/sdc /dev/sdd /dev/sde /dev/sdf /dev/sdg /dev/sdh"
```

Comment:
can you discover these dynamically instead of passing them as an arg ? please look at the nfsserver.sh script which contains that code portion

Reply:
Up to 10 drives will be automatically added now per OSS.